src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -96,6 +97,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case RedoCommand.COMMAND_ALIAS:
             return new RedoCommand();
 
+        case RemarkCommand.COMMAND_WORD:
```

Comment:
Add the ALIAS as well? To conform with other commands.

Reply:
yep next PR